lib/phlex/helpers.rb

```diff
@@ -27,7 +27,7 @@ def mix(*args)
 				in [String, String]
 					"#{old} #{new}"
 				else
-					new
+					new.nil? ? old : new
```

Comment:
What do you think about following the pattern matching style with something like:

```rb
case [old, new]
# ...
in [_, nil]
  old
else
  new
end
```

Reply:
the `_` was exactly what I was looking for. Thanks!